### ISSUE TITLE
Add Resolves/Closes Keywords to PR Template

### ISSUE THREAD

**Issue by DarhkVoyd** [json-schema-org/website#343]
#### Reason/Context

Currently, when a PR is created for an issue, the PR template includes `GitHub Issue:`, which only mentions the original issue in the PR.

It would be really helpful if the issues are connected to the PR, which will improve clarity and connect issues to PR. This way, when issues are mentioned in their PR, they will be connected, and the issues will be automatically closed upon merging the PR. Moreover, issues tab will display if a PR is created for the particular issue. 

#### Description

Introduce the use of `Resolves` or `Closes` keyword in the PR template. This improvement aims to establish a connection between mentioned issues and pull requests.

For reference:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Do you think this work might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

**Comment by benjagm** [comment]
Thanks for opening this issue @DarhkVoyd ! This makes lot of sense to me. 
@gregsdennis what is your view on this?

**Comment by aialok** [comment]
@benjagm, I would like to work on this issue. Please assign it to me.

It will be helpful in closing the issue. Adding "resolve" will enable automatic closure of the issue once the PR is merged.

Thank you!

**Comment by gregsdennis** [comment]
I think it's probably good to add as options.  I'd recommend having multiple options in a comment so that the opener can pick one that's appropriate.

- Resolves/Closes #___
- Related to #___ (for cases where the PR doesn't completely resolve an issue)
- Others?

**Comment by benjagm** [comment]
Can we add @gregsdennis requirements to the expected result?

**Comment by benjagm** [comment]
@aialok please go ahead!

**Comment by aialok** [comment]
> I think it's probably good to add as options. I'd recommend having multiple options in a comment so that the opener can pick one that's appropriate.
> 
> * Resolves/Closes #___
> * Related to #___ (for cases where the PR doesn't completely resolve an issue)
> * Others?
> 
> Might also be a good idea to put in a reminder (comment) that the [dependencies action](https://github.com/gregsdennis/dependencies-action) is available, so if another PR needs to be merged first, this one can be blocked until it is.

@gregsdennis Should I add [Dependencies Action Workflow](https://github.com/gregsdennis/dependencies-action) ?

**Comment by gregsdennis** [comment]
It's already org-wide

**Comment by aialok** [comment]
```md
<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**
-  Closes #___ <!-- Replace ___ with the issue number this PR resolves -->
-  Related to #___ <!-- Use when the PR doesn't completely resolve an issue -->
-  Others? <!-- Add any additional notes or references here -->


**Snapshots:**

<!--Add snapshots or videos wherever possible.-->

**If relevant, did you update the documentation?**

<!--Add link to it-->

**Summary**

<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
```

@gregsdennis If it is okay can we improve  `PR-Template` more by adding this to existing template?
